packages/parser-babel-config/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# 1.0.1 (March 7, 2018)
```

Comment:
Did we published this package already? If not you can remove this file and the other one for the rule as they will be created automatically.